### PR TITLE
Set CMake case

### PR DESCRIPTION
**Pull request by aprokop**

### CODE REVIEW

File: scripts/check_cmake_case.sh [hunk 1]
```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env bash
+
+# Based on https://cmake.org/gitweb?p=cmake.git;a=commitdiff;h=77543bd
+
+this_program=$(basename "$0")
+usage="Usage:
+  $this_program [options] -- check format of the C++ source files
```
Comment (dalg24) [inline scripts/check_cmake_case.sh]: uh?

**Comment by aprokop** [comment]
> Did you look into opportunity of code reuse between the original bash script and the new one you added?
> 
> Also have you looked at [cheshirekow/cmake_format](https://github.com/cheshirekow/cmake_format)

No and no.

File: scripts/check_cmake_case.sh (hunk 1 above)
Comment (aprokop) [reply scripts/check_cmake_case.sh]: oops, copy-paste mistake

**Comment by aprokop** [comment]
Whatever, not working on it.

**Pull request closed without merging**